### ISSUE TITLE
Handcuffs in wasteland

### ISSUE THREAD

**Issue by Soloknight** [micovery/ArmA3_Wasteland.Altis#48]
Hi Micovery,
I'm looking for a addon to include in wasteland from the rpg altislife. I would like a item included, handcuffs/addon on our server. Now at this time we only use the extDB so no saving is needed, unless we need it. We run some very pop servers and adding more soon. thanks

**Comment by micovery** [comment]
I like the idea of being able to restrain players. 

Players would be able to restrain other players when they are injured on the ground, and carry them around and put them into a vehicle (while restrained).

**Comment by IvanMMM** [comment]
I also have surrender script on my server. Now players could take items from surrendering player.

**Comment by Soloknight** [comment]
Surrender addon seems interesting for a fix until restraints, what does it exactly do, may I ask

**Comment by IvanMMM** [comment]
Well, we could use it as a base for this script. Just need to expand it a little. https://github.com/IvanMMM/ArmA3_Wasteland.Altis/tree/Development_main_addons%2Bsurrender

**Comment by Soloknight** [comment]
I tried this, and it seems to be real difficult to use, the player has to be stable right or down but not dead?

**Comment by IvanMMM** [comment]
Yes, basically this script adds surrender action. It will be strange to suggest to surrender to player who's already nearly dead. But we can add actions to incapacitated players too.

**Comment by micovery** [comment]
Ok ... say you are in "restrained" state ... the "Respawn", and "Abort" buttons should be disabled. That way the feeling really sinks in that they are not in control. 

What would be the penalty if you manage to exit the game while restrained? When you come back your stats are reset?

Also, if someone restrains you, and they leave you restrained for some time in the middle of no-where, you should be able to get out of the restrains.

Ah, and in order to prevent trolling, you should not be able to restrain members of your own team.
